canonical-web-and-design/jaas-dashboard
# Update the login UI to include a connecting state

## Done

Update the login UI to show a "connecting" state when we're connecting to the controller waiting to know if we've logged in yet or not. This always render the application under a mask to reduce the jarring effect of the 'login screen flash' in #79. 

@spencerbygraves The UX has been OK'd by @clagom but I just winged the design. Feel free to throw together some comments here or we can easily update this after it lands.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- It should show 'connecting' as it's waiting for a `visitURL` from the controller. 
- If you're already logged in it should then disappear and you should be logged in.
- If not it'll show a green button to log in. 
- Log in and then log out.
- You should get a green button again to log in.
- Log in and it should work

## Details

Fixes #79 
Fixes #76
